tests/jballerina-unit-test/src/test/resources/test-src/types/map/map-value.bal

```diff
@@ -1,3 +1,7 @@
+public type Animals Map;
+
+public type Map map<string>;
+
```

Comment:
Types can also be immediately before the test.

Reply:
Updated